plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -51,13 +51,14 @@ def show_listing(self, list_items, sort=None, content_type='episodes'):
         for title_item in list_items:
             list_item = xbmcgui.ListItem(label=title_item.title, thumbnailImage=title_item.art_dict.get('thumb'))
             url = self._url + '?' + urlencode(title_item.url_dict)
-            list_item.setProperty('IsPlayable', str(title_item.is_playable))
+            list_item.setProperty('isPlayable', 'true' if title_item.is_playable else 'false')
```

Comment:
setProperty is not case sensitive but in the kodi examples for setProperty they do start the values with a capital letter. 
from the docs => 

```
listitem.setProperty('AspectRatio', '1.85 : 1')
listitem.setProperty('StartOffset', '256.4')
```

Reply:
Yeah, I wasn't sure and this was just to test something. I looked at other plugins and most were doing it like this. I don't mind turning it back.

Beware this PR is a Work-In-Progress, so should not be merged :-) Some of the things do work correctly, but are not ready, e.g. the score/rating is nice, but can surpass 10, so I need to understand first what the value actually means... For programs the values are between 4.8 and 5.2, for episodes they range from 3 to 8, but I have seen values up to 38... (Undercover)